### ISSUE TITLE
Screen reader announcing "Group" on ‘Horizontal scroll bar’ present in ‘Web URL’ tab.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#119]
User Experience:

User who depend on screen reader might get confused and will not able to get proper information about the controls if screen reader announces as ‘group’ on controls.

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode.

4. Navigate to the Left nav and select Navigation button.

5. Navigate to the "Navigation" window and select add navigation button.

6. Navigate to the "Add navigation" dialog and select Hyperlink button .

7. Navigate to the various controls of the" Hyperlink" dialog box.

8. Navigate to ‘Web URL’ tab and select it.

9. Enable screen reader and navigate all the controls present in ‘Web URL’ tab.

 

Actual Result:

Screen reader announcing "Group" on ‘Horizontal scroll bar’ present in ‘Web URL’ tab.

 

Expected Result:

Screen reader should announce as ‘Horizontal scroll bar’ when screen reader focus moved on scroll bar control present in ‘Web URL’ tab.

 

Note:

Similar issue exist with Chrome + JAWS and Chrome + NVDA. 

 

MAS Reference:

MAS 1.3.1 - Info and Relationships (40)

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
"perfect-scrollbar" component issue.
